templates/default/nginx.yaml.erb

```diff
@@ -1,6 +1,12 @@
 instances:
 <% @instances.each do |i| -%>
   - nginx_status_url: <%= i['nginx_status_url'] %>
+    <%= "ssl_validation: #{i['ssl_validation']}" unless i['ssl_validation'].nil? -%>
```

Comment:
I waffled on whether to do it that way, but now that I think about it, what you put above will also be more consistent with the rest of the template. Let me take care of that.